admin/services/data-access/sce.data.service.js

```diff
@@ -37,15 +37,15 @@ sceDataService.sqlFindSceSchools = async () => {
   FROM ${sqlService.adminSchema}.[school]
   LEFT JOIN ${sqlService.adminSchema}.[sce]
     ON sce.school_id = school.id
-  WHERE school.leaCode = 702 
+  WHERE school.leaCode = 702
```

Comment:
I realise this is existing code - but why is there a magic number in there?

Reply:
that's bizarre!

Reply:
@GeorgeChatzigiannis any idea on this one?

Reply:
seems like all the sce schools have a lea code of 702 as per https://github.com/DFEAGILEDEVOPS/MTC/pull/1064/files

Aside from the magic number I don't really like the naming convention used on this service as well.

Reply:
ok, ill refactor that out to a variable to make it obvious.  thanks